content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/moves_history.rst

```diff
@@ -0,0 +1,100 @@
+=============
+Moves history
+=============
+
+The *Moves History* report in Odoo *Inventory* provides a detailed record of product movements
+(containing past and current locations), lot numbers, and reasons for movement. Reports can be
+generated for any time frame, making this report essential for analyzing stock levels, monitoring
+inventory turnover, and identifying any discrepancies in inventory.
+
+.. note::
+   The reporting feature is only accessible to users with :doc:`admin access
+   <../../../../general/users/access_rights>`.
+
+To access the stock report, go to :menuselection:`Inventory app --> Reporting --> Moves History`.
+
+.. image:: moves_history/moves-history.png
+   :align: center
+   :alt: Display Moves History report accessible from Inventory > Reporting > Moves History.
+
+.. _inventory/warehouses_storage/moves-history-report:
+
+Navigate the moves history report
+=================================
+
+In the report, the columns represent:
+
+- :guilabel:`Date`: calendar date and time of the stock move.
+- :guilabel:`Reference`: description of the reason for the stock move or quantity change, such as a
+  receipt number (e.g. `WH/IN/00012`).
+- :guilabel:`Product`: name of the product involved in the move.
+- :guilabel:`Lot/Serial Number`: specifies the lot or serial number of the tracked product being
+  moved.
+- :guilabel:`From`: source location of the moved product.
+- :guilabel:`To`: destination location of the moved product.
+- :guilabel:`Quantity`: number of products moved.
+- :guilabel:`Unit`: unit of measure of the products moved.
+- :guilabel:`Status`: indicates the move status, which can be :guilabel:`Done`,
+  :guilabel:`Available` (ready for action), or :guilabel:`Partially Available` (insufficient
+  quantities to complete the operation).
+
+Search options
+--------------
+
+Use the following search options to customize the :guilabel:`Moves History` report to display
+relevant information
+
+.. tabs::
+
+   .. tab:: Filters
+
+      The :guilabel:`Filters` section allows users to search among pre-made and custom filters to
+      find specific stock records.
+
+      - :guilabel:`To Do`: show stock move records that are in progress. This includes lines with a
+        :guilabel:`Status` column value of :guilabel:`Available` or :guilabel:`Partially Available`.
+      - :guilabel:`Done`: completed stock moves, with a :guilabel:`Status` of :guilabel:`Done`.
+      - :guilabel:`Incoming`: displays move records from vendor locations.
+      - :guilabel:`Outgoing`: displays move records to customer locations, including customer
+        returns.
+      - :guilabel:`Internal`: displays move records from one internal location to another.
+      - :guilabel:`Manufacturing`: shows records where products were produced from the virtual,
+        production location.
+
+      .. seealso::
+         :doc:`../inventory_management/use_locations`
```

Comment:
Consider embedding this a little further in, so it's inline with the **Manufacturing** list item which references it
```suggestion
        .. seealso::
           :doc:`../inventory_management/use_locations`
```